### PR TITLE
blog(embertimes138): ember modifiers blog post

### PR DESCRIPTION
**Pull request by chrisrng**

### CODE REVIEW

File: source/2020-03-06-the-ember-times-issue-138.md [hunk 1]
```diff
@@ -22,13 +23,23 @@ READMORE
...
+{{!-- my-component.hbs --}}
+<button
+  {{key-down this.handleEnter key='Enter'}}
+  My Button
+</button>
+```
+
+Unlike attributes, element modifiers [do not run on the server-side](https://guides.emberjs.com/release/components/template-lifecycle-dom-and-modifiers/#toc_event-handlers), because there is no general-purpose and complete way to serialize the results of an element modifier into HTML.
+
+Be sure to check out [ember-render-modifiers](https://github.com/emberjs/ember-render-modifiers) and [ember-focus-trap](https://github.com/josemarluedke/ember-focus-trap) as well and read the [full blog](https://gaiety.life/introducing-ember-modifiers) on Ember Modifiers!
```
Comment (jayjayjpg) [inline source/2020-03-06-the-ember-times-issue-138.md]: I think it'd be 

```suggestion
Be sure to check out [ember-render-modifiers](https://github.com/emberjs/ember-render-modifiers) and [ember-focus-trap](https://github.com/josemarluedke/ember-focus-trap) as well and read the [full article](https://gaiety.life/introducing-ember-modifiers) on Ember Modifiers!
```
What do you think? I'm a non-native speaker though, so I'm not sure if `blog` applies here after all - feel free to use the wording that works best in your opinion!

**Review by jayjayjpg — APPROVED** [review]
This reads great! I left one more wording suggestion, but otherwise this i already good to go - thank you for another :sparkles: writeup this week! :rainbow:

File: source/2020-03-06-the-ember-times-issue-138.md [hunk 2]
```diff
@@ -22,13 +23,23 @@ READMORE
...
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+[Ava Gaiety Wroten (@sharpshark28)](https://github.com/sharpshark28) wrote a blog post on the [ember-modifier](https://github.com/ember-modifier/ember-modifier) addon which provides an API for authoring [element modifiers](https://blog.emberjs.com/2019/03/06/coming-soon-in-ember-octane-part-4.html) in Ember.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+We learn how to use the [`{{on}}` modifier](https://github.com/emberjs/rfcs/blob/master/text/0471-on-modifier.md) to track the focus state of a DOM element. The blog then describes how to handle key presses by creating our very own modifier such as the example `{{key-down}}` below:
+
+```
```
Comment (jayjayjpg) [inline source/2020-03-06-the-ember-times-issue-138.md]:
```suggestion
```handlebars
```

This should resolve the markdown linter error on the CI

File: source/2020-03-06-the-ember-times-issue-138.md (hunk 1 above)
Comment (chrisrng) [reply source/2020-03-06-the-ember-times-issue-138.md]: I think both is fine but i'll defer to you!

**Pull request merged**